test/presenters/taxon_presenter_test.rb

```diff
@@ -49,4 +52,150 @@
       TaxonPresenter.new(taxon).accordion_content
     end
   end
+
+  describe 'options' do
+    let(:content_hash) { funding_and_finance_for_students_taxon }
+    let(:content_item) { ContentItem.new(content_hash) }
+    let(:taxon) { Taxon.new(ContentItem.new(content_hash)) }
+    let(:child_taxon) { taxon.child_taxons.first }
+    let(:taxon_presenter) { TaxonPresenter.new(taxon) }
+
+    describe '#options_for_leaf_content' do
+      before :each do
+        @search_results = generate_search_results(15)
+        stub_content_for_taxon([taxon.content_id], @search_results)
+      end
```

Comment:
Picky maybe, but could there be a newline between each block? It's hard to read such a dense wall of text